cellx/tools/projection.py

```diff
@@ -125,8 +142,6 @@ def __call__(
 
             imgrid[blockx, blocky, :] = im
 
-        # return the extent, i.e. the mapping back to the components of the
-        # manifold
         extent = [min(xe), max(xe), min(ye), max(ye)]
```

Comment:
I wonder if we should change this to a tuple?

Reply:
Good idea, I'll add another commit!